lib/compiler/ops.ts

```diff
@@ -54,6 +54,16 @@ export class Compute extends PointWiseOp {
         return `PointWiseOp.Compute(${this.expression} as ${this.alias})`;
     }
 }
+
+export class Verification extends PointWiseOp {
```

Comment:
Why is this called "Verification"?

Reply:
it's verifying if a condition is true or not for a table? I just thought `Boolean` or `BooleanExpression` would be a weird here

Reply:
Maybe BooleanComputation or similar? That's what it is anyway. Verification is weird, it sounds like you're enforcing some assertion.